docs/openstack/edpm_adoption.md

```diff
@@ -305,3 +294,48 @@ done
     ```
     oc wait --for condition=Ready osdpns/openstack --timeout=30m
     ```
+
+* Manually complete the Nova Compute services upgrade from Wallaby to
```

Comment:
@gibi @SeanMooney It seems we need more than just these steps, plus nova compute Antelope containers running? I followed the below steps, and got back the error message:
```
nova.exception.TooOldComputeService: Current Nova version does not support computes older than Yoga...
```

Reply:
@jistr @SeanMooney I suggest keeping the FFU flag ON for this simple nova services adoption scenario.
We need to figure multi-cell and FFU later.

Reply:
After the EDPM computes are adopted and therefore upgraded to antelope you need to restart the control plane services to allow the recalculation of the minimum compute service version in the cluster. Please check the 3 point from https://docs.openstack.org/nova/latest/admin/upgrades.html#rolling-upgrade-process

Reply:
this needs to be done before unpatching the FFU configs? I'll try that, thanks!

Reply:
I think we default to `[upgrade_levels]compute=auto` so the important thing is restart all services with the new code and then nudge the control plane to recalculate the min compute service version, which will be high enough at that point to continue without the FFU WA config.